### PR TITLE
[CESQL] Added suggestions sections with error handling

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Followup of #809

Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

### CODE REVIEW

File: expression-language.md [hunk 1]
```diff
@@ -411,7 +415,23 @@ sequence = 10
 `Integer x Integer`. Because the right parameter of the operator is an _Integer_ and there is only one `=` definition
 which uses the type _Integer_ as right parameter, `sequence` is casted to _Integer_.
 
-## 4. Examples
+## 4. Suggestions
```
Comment (duglin) [inline expression-language.md:418]: Maybe "Implementation Guidance" ?

Comment (slinkydeveloper) [reply expression-language.md:418]: _Implementation suggestions_?

**Comment by duglin** [comment]
Approved on the 5/20 call with the suggested section heading change.
@slinkydeveloper let me know when that's done and I'll merge

**Comment by slinkydeveloper** [comment]
@duglin is it ok now?

**Comment by duglin** [comment]
@slinkydeveloper thanks,merging

**Pull request merged**